readme.md

```diff
@@ -104,12 +104,13 @@ special file ending. Several file endings have been discussed, in the end
 
 [more here](https://github.com/nodejs/node/wiki/ES6-Module-Detection-in-Node#option-2-new-file-extension-for-es6-modules)
 
-### 3) ~~Content-Sniffing~~ _(rejected)_
+### 3) ~~Content-Sniffing aka. Double Parsing~~ _(rejected)_
 
 The content would be pre-parsed to identify whether its a `CommonJS` or
 `ES2015` module.
 
-_(Rejected because the detection is fuzzy and could lead to user confusion and problems with tooling)_
+_(Rejected because the detection has ambiguity in `CommonJS` packages. This could be revisited if the TC39 changes the specification of ES2015 modules but
+could still be rejected because of performance reasons)_
```

Comment:
it could also still be rejected because not every environment that needs to disambiguate module vs script has a JS parser available.